software/hi-events.yml

```diff
@@ -0,0 +1,16 @@
+name: "Hi.Events"
+website_url: "https://hi.events"
+source_code_url: "https://github.com/HiEventsDev/hi.events"
+description: "An open-source, feature-packed event management and ticketing platform for conferences, concerts, and more. Offering customizable event pages and embeddable ticket widgets."
+licenses:
+  - AGPL-3.0
+platforms:
+  - JavaScript
+  - Nodejs
+  - PHP
+  - Docker
+tags:
+  - Booking and Scheduling
+  - E-commerce
+depends_3rdparty: false
```

Comment:
```suggestion
```
This is already set to false, no need for this line 👍🏻

Reply:
Updated 👍